### DISCUSSION TITLE
Logo Pasteleria. Pagina o app

### DISCUSSION THREAD

**Discussion by Eholguin05** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#305] [Q&A] [answered]
Que tal amigos, ando muy atrasado y entiendo si no pueden apoyar, me gustaria saber si me pueden ayudar en decirme que pagina usaron para el logo de la pasteleria?

Por su tiempo, gracias!

**Comment by GloriaCoralCerecedo** [comment]
Hola, ¿Para para realizar el logo?

**Reply by Eholguin05** [reply]
si

**Comment by martinGM05** [comment, marked as answer]
Use Power Point jeje, con power le quitas fondo y le agregas un texto y listo jeje

**Reply by Eholguin05** [reply]
Gracias

**Comment by AngelCruzO** [comment]
Con canva

**Comment by doguedogue** [comment]
Exacto Canva como con el logo de la taqueria, hay un vídeo de eso

https://www.canva.com/es_419/

👍🏼
